src/core/util/Integer.ts

```diff
@@ -101,6 +101,10 @@ export default class Integer {
     return i & 0x3f;
   }
 
+  public static truncDivision(i: number): number {
+    return Math.trunc(i);
```

Comment:
Oh I thought this would be a method for performing the division, like:

```typescript
  public static truncDivision(a: number, b: number): number {
    const result = a / b;
    return Math.trunc(result);
  }
```

I see no reason for just wrapping `Math.trunc` inside another function that will aggregate no functionality.

Reply:
yep, I thought about that while doing. Ok, I will change that one.